Cannot Install NPM Packages

I'm attempting to clone this project and run the examples, the following error is repeatedly occurring:

```
ember-simple-auth [master] % npm install

> phantomjs@1.9.17 install /Users/james/Projects/ember-simple-auth/node_modules/grunt-mocha/node_modules/grunt-lib-phantomjs/node_modules/phantomjs
> node install.js

PhantomJS detected, but wrong version 2.0.0 @ /usr/local/bin/phantomjs.
Download already available at /var/folders/bj/d0jc0q090k3bg9lwlpltyt6r0000gn/T/phantomjs/phantomjs-1.9.8-macosx.zip
Extracting zip contents
Removing /Users/james/Projects/ember-simple-auth/node_modules/grunt-mocha/node_modules/grunt-lib-phantomjs/node_modules/phantomjs/lib/phantom
Copying extracted folder /var/folders/bj/d0jc0q090k3bg9lwlpltyt6r0000gn/T/phantomjs/phantomjs-1.9.8-macosx.zip-extract-1433882784415/phantomjs-1.9.8-macosx -> /Users/james/Projects/ember-simple-auth/node_modules/grunt-mocha/node_modules/grunt-lib-phantomjs/node_modules/phantomjs/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /Users/james/Projects/ember-simple-auth/node_modules/grunt-mocha/node_modules/grunt-lib-phantomjs/node_modules/phantomjs/lib/phantom/bin/phantomjs
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! path /Users/james/Projects/ember-simple-auth/node_modules/grunt-lintspaces/node_modules/editorconfig/bin\editorconfig
npm ERR! code ENOENT
npm ERR! errno -2

npm ERR! enoent ENOENT, chmod '/Users/james/Projects/ember-simple-auth/node_modules/grunt-lintspaces/node_modules/editorconfig/bin\editorconfig'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/james/Projects/ember-simple-auth/npm-debug.log
```

npm-debug.log is here https://gist.github.com/jherdman/0a6b62629d716b6850b3

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "grunt-string-replace": "^0.2.7",
     "yuidocjs": "^0.3.49",
     "grunt-compile-handlebars": "^0.6.3",
-    "grunt-lintspaces": "^0.4.1",
+    "grunt-lintspaces": "0.7.0",
     "connect-redirection": "^0.0.1",
     "body-parser": "^1.0.0",
     "handlebars": "^2.0.0",

